src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -83,16 +82,13 @@ class ProjectConfigurationModal extends Component<Props, State> {
   };
 
   render() {
-    const { project, hideModal, isVisible, queueEmpty } = this.props;
+    const { hideModal, isVisible, dependenciesChangingForProject } = this.props;
     const { activeField } = this.state;
     const { projectIcon } = this.state;
-    const { name } = project || { name: '' };
 
     return (
       <Modal isVisible={isVisible} onDismiss={hideModal}>
-        <ModalHeader title="Project settings">
-          <Description>Change the settings of project {name}</Description>
```

Comment:
I removed this because I didn't think it was necessary (it does mean the user can't see what the current project name is, if they rename it, but I don't think they need to)